### PR TITLE
Sign in page creation

### PR DESCRIPTION
**Pull request by martadiachok**
Add page object SignInPage, SignUpPage and page component Calendar

### CODE REVIEW

File: src/main/java/com/easyrest/components/Calendar.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class Calendar {
+    protected WebDriver driver;
+    public Calendar(WebDriver driver) {
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/Calendar.java:11]: Java Code Conventions, constructor after variables

File: src/main/java/com/easyrest/components/Calendar.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        this.driver = driver;
+    }
+
+    private By chooseYearButton = By.xpath("//*[text()='2022']");
+    private By yearList = By.xpath("//div[@role='button']");
+    private By monthName = By.xpath("//*[@class='MuiTypography-root-41 MuiTypography-body1-50 MuiTypography-alignCenter-64']");
+    private By switchMonthLeftButton = By.xpath("/html/body/div[2]/div[2]/div/div[1]/div[2]/div[1]/button[1]");
+    // private final By switchMonthLeftButton = By.xpath("//*[@class='MuiButtonBase-root-105 MuiIconButton-root-337 MuiPickersCalendarHeader-iconButton-334']");
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/Calendar.java]: Brittle (fragile) locator

File: src/main/java/com/easyrest/components/Calendar.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import java.util.List;
+
+public class Calendar {
+    protected WebDriver driver;
+    public Calendar(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    private By chooseYearButton = By.xpath("//*[text()='2022']");
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/Calendar.java:16]: Java Code Conventions, constructor after variables

**Review by ViktoriaMilkevich — APPROVED** [review]

File: src/main/java/com/easyrest/common/CommonAction.java [hunk 4]
```diff
@@ -0,0 +1,2 @@
+package com.easyrest.common;public class CommonAction {
```
Comment (dmatsev) [inline src/main/java/com/easyrest/common/CommonAction.java]: do you need this class?

File: src/main/java/com/easyrest/components/Calendar.java [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        this.driver = driver;
+    }
+    
+    protected WebDriver driver;
+
+    private By chooseYearButton = By.xpath("//*[text()='2022']");
+    private By yearList = By.xpath("//div[@role='button']");
+    private By monthName = By.xpath("//*[@class='MuiTypography-root-41 MuiTypography-body1-50 MuiTypography-alignCenter-64']");
+    private By switchMonthLeftButton = By.xpath("/html/body/div[2]/div[2]/div/div[1]/div[2]/div[1]/button[1]");
+    // private final By switchMonthLeftButton = By.xpath("//*[@class='MuiButtonBase-root-105 MuiIconButton-root-337 MuiPickersCalendarHeader-iconButton-334']");
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/Calendar.java]: Do you need this commented code?

**Review by dmatsev — APPROVED** [review]
Awesome! Great job!

**Pull request merged**